tutorial/practicals/nemo/4_nemo_openacc/README.md

```diff
@@ -0,0 +1,417 @@
+# Using PSyclone to add OpenACC to NEMO - Tutorial 4 #
```

Comment:
Now that you have made the Makefile not compile by default in this section of the tutorial it would probably be worth pointing out that by doing make myexe it will compile - I might have missed that bit of information - if so apologies.

Reply:
Done.